Makefile

```diff
@@ -35,3 +41,89 @@ build-dependencies-axolotl:
 clean:
 	rm -f axolotl
 	rm -rf axolotl-web/dist
+
+check_platform_arm64:
+	@echo "Building Axolotl for Mobian (aarch64/amd64)"
+  ifneq ($(shell uname),Linux)
+	@echo "Platform unsupported - only available for Linux" && exit 1
+  endif
+  ifneq ($(shell uname -m),aarch64)
+	@echo "Machine unsupported - only available for aarch64/arm64" && exit 1
+  endif
+  ifneq ($(shell which apt),/usr/bin/apt)
+	@echo "OS unsupported - apt not found" && exit 1
+  endif
+
+dependencies_arm64:
+	@echo "Installing dependencies for building Axolotl..."
+	@sudo apt update
+	@sudo apt install golang nodejs npm python
+
+build_arm64:
+	@echo "Downloading (go)..."
+	@cd $(WORKDIR) && go mod download
+	@echo "Installing (npm)..."
+	@cd $(WORKDIR)/axolotl-web && npm install
+	@echo "node-sass does not support aarch64/amd64 so it has to be rebuilt"
+	@echo "Rebuilding of npm-sass..."
+	@cd $(WORKDIR)/axolotl-web && npm rebuild node-sass
+	@echo "Building (npm)..."
+	@cd $(WORKDIR)/axolotl-web && npm run build
+	@mkdir -p $(WORKDIR)/build/linux-arm64/axolotl-web
+	@echo "Building (go)..."
+	@cd $(WORKDIR) && env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl .
+	@cp -r axolotl-web/dist build/linux-arm64/axolotl-web
+	@cp -r guis build/linux-arm64
+	@echo "Building complete."
+
+prebuild_package_arm64: package_clean_arm64
+	@echo "Prebuilding Debian package..."
+# Get the source tarball
+	@cd $(WORKDIR) && wget https://github.com/nanu-c/axolotl/archive/main.tar.gz
```

Comment:
I cant seem to find this .tar.gz file in the repo. Am I missing something, or how does this work?

Reply:
The main tarball of the repo is renamend in the next step of the Makefile, because debmake needs the source to continue packaging.

Reply:
From what I can tell, this downloads a tar.gz archive of this repo. 

As this is downloading based on the `main` branch, this build will not be deterministic/reproducible.

Instead, what about using the most recent git release tag - the URL would be `https://github.com/nanu-c/axolotl/archive/v0.9.9.tar.gz`. This pulls a fixed set of files, and has the additional benefit that we could also verify the downloaded archive against a sha256sum for example.

Reply:
Let me have a look into the Debian packaging docs again to see, which tarball is standard as source reference... I'll have to get in touch with some of the Debian/Mobian packaging pros for that, too.